### DISCUSSION TITLE
Pobieranie informacji o zamówieniach z kont powiązanych

### DISCUSSION THREAD

**Discussion by jakubmaguza** [allegro/allegro-api#4848] [Suggestions - rejected]
Hej,
W panelu allegro jest możliwość sprawdzenia zamówień z kont powiązanych bez przelogowywania się.
Czy możecie dodać taką opcję w API?

Dobrego dnia!

**Comment by Onixarts** [comment]
Przecież do RESTa się nie logujesz, tylko wysyłasz odpowiednie żądanie ;). Jeśli cokolwiek by było robione w tej kwestii to bardzo proszę jako opcja, żeby zachować wsteczną kompatybilność. Zwracanie wyników z różnych kont za pomocą requesta do konta X mocno naruszy istniejące rozwiązania.

**Reply by jakubmaguza** [reply]
Chodzi o to, że jeśli firma X ma 10 kont na allegro, to dobrze by było móc pobrać dane z wszystkich 10 kont, a nie powiązywać wszystkie konta oddzielnie. Wtedy w zapytaniu wystarczyłoby dodać login/id konta, o które chcemy zapytać.

W przypadku braku loginu/id oczywiście zapytanie powinno zostać wysłane do konta domyślnego.

**Comment by MartaNowaczyk** [comment]
@jakubmaguza na ten moment nie planujemy dodawać takiej funkcjonalności, dlatego oznaczam sugestię jako odrzuconą.
